pom.xml

```diff
@@ -679,10 +679,13 @@
             <!-- License Files for other licenses -->
             <exclude>**/UNLICENSE</exclude>
             <!--
-              Explicitly exclude this file as I was unable
-              to add a new license to the configuration.
+              Explicitly exclude these files as I was unable
+              to add a new licenses to the configuration.
             -->
             <exclude>**/Expression.g4</exclude>
+            <exclude>**/KeyStoreLoader.java</exclude>
+            <exclude>**/OPCUAServer.java</exclude>
+            <exclude>**/Plc4xNamespace.java</exclude>
```

Comment:
Actually you shouldn't need these exclusions. The rat tool should detect the EPL header. But it doesn't harm either.